### DISCUSSION TITLE
no samples for update or delete functionality

### DISCUSSION THREAD

**Discussion by nshathish** [Azure/azure-functions-sql-extension#519] [General]
why is there no samples for update or delete functionality

**Comment by Charles-Gagnon** [comment]
For update you just use an output binding with specifying the identity column - if it already exists it will update that row with the new values. 

https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/BindingsGuide_Dotnet.md#sqlattribute-for-output-bindings

And we actually have a sample for that : https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-csharp/OutputBindingSamples/AddProductWithIdentityColumnIncluded.cs

Delete isn't something we have samples for since that isn't currently a primary feature we're supporting. But you should be able to accomplish this by using an input binding and having the query do a delete instead of a select. You can look at https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-csharp/InputBindingSamples/GetProducts.cs and then just have the query be something like `DELETE FROM dbo.Foo WHERE Id= @Id`

I created an issue to track adding delete samples to the repo to help other people trying to figure this out as well. https://github.com/Azure/azure-functions-sql-extension/issues/520
